ledger/ledger-api-bench-tool/src/test/lib/scala/com/daml/ledger/api/benchtool/submission/ActiveContractsObserver.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool.submission
+
+import com.daml.ledger.api.benchtool.util.ObserverWithResult
+import com.daml.ledger.api.v1.active_contracts_service.GetActiveContractsResponse
+import org.slf4j.{Logger, LoggerFactory}
+
+import scala.concurrent.Future
+
+object ActiveContractsObserver {
```

Comment:
Are these just moved or with some actual changes?
(Also consider making pure refactor changes into a separate PR, possibly to be merged before this one)